scripts/date-logic/remove-isfutureonly-key.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable */
+
+// Number of forms with isFutureOnly key - expect 0 after running update
+
+db.forms.count({
+  form_fields: { $elemMatch: { isFutureOnly: { $exists: true } } },
+})
+
+// Number of form fields with isFutureOnly key - expect 0 after running update
+
+const formFieldsWithIsFutureOnlyExist = [
+  {
+    $match: {
+      form_fields: { $elemMatch: { isFutureOnly: { $exists: true } } },
+    },
+  },
+  { $project: { form_fields: 1 } },
+  { $unwind: '$form_fields' },
+  { $match: { 'form_fields.isFutureOnly': { $exists: true } } },
+  { $count: 'numFormFields' },
```

Comment:
dangling comma will throw error here
```suggestion
  { $count: 'numFormFields' }
```

Reply:
edited